### ISSUE TITLE
Weight 400 bug at web online Roboto font family from google fonts

### ISSUE THREAD

**Issue by amaz0mbie** [google/roboto#151]
I use Roboto font on my website as body text, linked to google fonts library. Everything was just fine, but on the last weeks I noticed, that my body font (400) goes bolder. After I've checked google font url, and there is the same. The weight 400 type is more or less the double as bold as it should, somewhere between 700-and 900.
https://www.google.com/fonts

Am I in the good place to send bug report? Could you please fix it? Im waiting for your kind reply, thanks.

**Comment by davelab6** [comment]
Please uninstall the fonts as a temporary workaround

**Comment by amaz0mbie** [comment]
thanks Dave for your quick reply, but should I wait for a bug fix in the near future?

**Comment by jamesgk** [comment]
I'm not seeing this bug here, so my guess is that the font should work fine for visitors on your website.

When you had the font installed locally, where had you gotten it from? What operating system / browser were you using?

**Comment by amaz0mbie** [comment]
Oh, its a good news, maybe the bug in my local OS.

I did not installed Roboto font locally, I only installed from Adobe typekit Robot Bold, which was free, maybe this is the reason why I see all of my browser in Bold the body texts?

I see the issue on my macbook, el capitan, all of my browser safari, chrome, firefox.

What is the solution to have the complete Roboto font kit on my computer with Europen CE fonts like Ő, Ű, etc.. could I dowload and install the complete font kit to my computer free from google fonts or else where?

Thanks for your comment, it helped a lot!
